apps/test/unit/lib/levelbuilder/script-editor/UnitEditorTest.jsx

```diff
@@ -425,9 +425,54 @@ describe('UnitEditor', () => {
       $.ajax.restore();
     });
 
+    it('saves successful if unit is not a course and only version year is set', () => {
```

Comment:
nit: successfully (sorry, there is just a lot going on in this sentence so every bit helps to make it more readable)